src/libaktualizr/uptane/fetcher.cc

```diff
@@ -23,8 +23,8 @@ bool Fetcher::fetchRole(std::string* result, int64_t maxsize, RepositoryType rep
 static size_t DownloadHandler(char* contents, size_t size, size_t nmemb, void* userp) {
   assert(userp);
   auto* ds = static_cast<Uptane::DownloadMetaStruct*>(userp);
-  uint64_t downloaded = size * nmemb;
-  uint64_t expected = ds->target.length();
+  size_t downloaded = size * nmemb;
```

Comment:
Why changing the type? I hoped we still could be able to download images larger than 4Gb on 32 bit devices. There are no tests for that right now, but let's not break that deliberately.

Reply:
OK, I'll remove the last commit in the branch